client/notifications/socket-handlers.ts

```diff
@@ -1,7 +1,9 @@
 import type { NydusClient, RouteHandler, RouteInfo } from 'nydus-client'
-import { NotificationEvent } from '../../common/notifications'
+import { NotificationEvent, NotificationType } from '../../common/notifications'
 import { dispatch, Dispatchable } from '../dispatch-registry'
 
+const ELECTRON_ONLY_NOTIFICATION_TYPES = [NotificationType.PartyInvite]
```

Comment:
You should provide an explicit type for this array, TS generally isn't gonna be able to pick a good one (and you should also make it `Readonly` so that we don't accidentally mutate it

Reply:
Honestly it's probably better to just make this a `Set` too. It doesn't matter that much with the amount of types we're talking about here, but may as well pay the one-time init cost to get O(n) inclusion checks